tools/roll_release.sh

```diff
@@ -103,8 +103,12 @@ fi
 
 echo ""
 echo "Tagging release."
-git tag --annotate "${VERSION_FULL_TAG}" --message="${VERSION_TITLE}"
-git push origin "${VERSION_FULL_TAG}"
+if [[ "$(git tag -l ${VERSION_FULL_TAG})" == "" ]]; then
```

Comment:
Actually, does this check if the tag exists locally or on the remote? I assume locally only. Could the push to `origin` be done even if the tag has already been created (and maybe also pushed)?

Reply:
Thanks! Yeah, wouldn't have been a big deal as it was, but the update looks even better :)